pulp_smash/tests/pulp3/constants.py

```diff
@@ -22,17 +22,17 @@
 FILE_PUBLISHER_PATH = urljoin(BASE_PUBLISHER_PATH, 'file/')
 
 IMPORTER_DOWN_POLICY = {'background', 'immediate', 'on_demand'}
```

Comment:
Same. I'm OK with leaving this name as is, so long as an issue is created in Smash's issue tracker.